### DISCUSSION TITLE
美国 | 连波的闲谈杂鱼

### DISCUSSION THREAD

**Discussion by giscus** [godruoyi/gblog#29] [Announcements]
# 美国 | 连波的闲谈杂鱼

昨天晚上开了一个小时的会，受益良多；以下只是我的感悟，请不要信以为真。

https://godruoyi.com/posts/america/

**Comment by herijaya** [comment]
Hello and greetings, fellow bloggers! This is a warm welcome from MAXsi ID Blog. Visit us at [MAXsi ID Blog](https://blog.maxsi.id).

**Reply by godruoyi** [reply]
Your blog looks pretty cool. Do you have any open-source plan for it?

**Comment by 3maoyap** [comment]
Tensorflow是框架哈，不是模型

**Reply by godruoyi** [reply]
谢谢纠正，👍

**Comment by TheSaltwaterRoom** [comment]
我哭了，我也想润

**Comment by GoodBoyboy666** [comment]
环境是很重要的
